### PR TITLE
fix: [PL-28465]: update message in user group page to show linked sso ldap sync of users message

### PR DESCRIPTION
**Pull request by pbarapatre10**
#### Screenshots
Before: 
image

After:
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11471)

**Comment by pbarapatre10** [comment]
retrigger jest

**Comment by pbarapatre10** [comment]
retrigger coverage

**Review by upadhyay-an — DISMISSED** [review]

**Comment by pbarapatre10** [comment]
retrigger coverage

**Comment by pbarapatre10** [comment]
retrigger jest

**Comment by pbarapatre10** [comment]
retrigger coverage

**Comment by pbarapatre10** [comment]
trigger cypress

**Comment by pbarapatre10** [comment]
trigger cypress

**Comment by pbarapatre10** [comment]
retrigger jest

File: src/modules/20-rbac/pages/UserGroupDetails/views/MemberList.tsx [hunk 1]
```diff
@@ -248,7 +249,11 @@ const MemberList: React.FC<MemberListProps> = ({ ssoLinked, userGroupInherited,
         />
       </Container>
     )
-  return <NoDataCard icon="nav-project" message={getString('rbac.userDetails.noMembersMessage')} />
+  return linkedSSOType && linkedSSOType === AuthenticationMechanisms.LDAP ? (
+    <NoDataCard icon="nav-project" message={getString('rbac.userDetails.userGroup.linkedSSOLdapUsersMessage')} />
```
Comment (reetika04) [inline src/modules/20-rbac/pages/UserGroupDetails/views/MemberList.tsx:253]: You can shift this condition to just message.

Comment (pbarapatre10) [reply src/modules/20-rbac/pages/UserGroupDetails/views/MemberList.tsx]: Done! thanks

**Review by reetika04 — APPROVED** [review]

**Review by upadhyay-an — APPROVED** [review]

**Pull request merged**